packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js

```diff
@@ -5760,6 +5760,48 @@ QUnit.module('datebox validation', {}, () => {
     });
 });
 
+[
```

Comment:
we have separate file for masked datebox tests. I suggest moving test to datebox.mask.tests. Also, do we really need to cretea a separate module for 1 test?